Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.py

```diff
@@ -181,24 +184,26 @@ def storage_account_create_update_request(self, args: Dict) -> Dict:
 
         return self.ms_client.http_request(
             method='PUT',
-            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
-                       f'/{args["account_name"]}',
-            params={
-                'api-version': API_VERSION,
-            },
+
+            full_url=f'https://management.azure.com/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/providers/Microsoft.Storage/\
+storageAccounts/{account_name}?api-version={API_VERSION}',
             json_data=json_data_args,
             resp_type='response'
         )
 
-    def storage_blob_service_properties_set_request(self, args: Dict) -> Dict:
+    def storage_blob_service_properties_set_request(self, subscription_id: str | None,
+                                                    resource_group_name: str | None, args: Dict) -> Dict:
```

Comment:
same question about the args and the account_name.

Reply:
same here:)